recipe/files.php

```diff
@@ -25,5 +25,30 @@ function () {
 task(
     'sumo:files:put',
     function () {
+        // ask for confirmation
+        if (!askConfirmation('Are you sure? This will overwrite files on production!')) {
+            return;
+        }
+
+        $sharedDirectories = get('shared_dirs');
+        if (!is_array($sharedDirectories) || empty($sharedDirectories)) {
+            return;
+        }
+
+        // remove some system dirs
+        $directoriesToIgnore = [
+            'var/log',      // this directory may contain useful information
+            'var/sessions'  // this directory may contain active sessions
```

Comment:
missing trailing comma

Reply:
Fixed in 6f87af62fa9773244bedf05eb86b45bf08ebb413